bump_version.sh

```diff
@@ -27,6 +29,10 @@ else
       tmp_file=/tmp/version.$$
       sed "s/$old_version_regex/$new_version/" $VERSION_FILE > $tmp_file
       mv $tmp_file $VERSION_FILE
+      sed "s/$old_version_regex/$new_version/" > $FLAKE_FILE
+      # Run flake update to update the flake.lock file
+      nix flake update
+      git add $FLAKE_FILE $FLAKE_LOCK_FILE
       git add $VERSION_FILE
```

Comment:
I don't see a reason not to combine these two lines:
```suggestion
      git add $FLAKE_FILE $FLAKE_LOCK_FILE $VERSION_FILE
```

Reply:
@jsf9k Please see https://github.com/cisagov/awssh/pull/27/commits/0ab4742c67b6c8239dcfc73ae13ef0fd58e9ec45